### DISCUSSION TITLE
移动端的react 是指的rn吗，rn可以用这个框架吗？

### DISCUSSION THREAD

**Discussion by Transformi** [Tencent/tdesign#372] [Q&A]
移动端的react 是指的rn吗，rn可以用这个框架吗？

**Comment by ZHUBoer** [comment]
> 移动端的react 是指的rn吗，rn可以用这个框架吗？

不是 rn，就是 react，版本为 17。rn 应该可以用吧，React Native 从 0.64 版本开始支持 React 17。
